### PR TITLE
Remove tests.js from the default --initkarma behavior

### PR DESCRIPTION
**Pull request by nickpape-msft**
Instead, we should use a glob pattern in the karma file. The tests.js file is funky and looks unprofessional.

### CODE REVIEW

**Comment by pgonzal** [comment]
Awesome -- can't believe this was such a simple fix.  :-)

File: gulp-core-build-karma/karma.config.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = function(config) {
 
 
     // list of files / patterns to load in the browser
-      files: [bindPolyfillPath].concat(['lib/tests.js']),
+      files: [bindPolyfillPath].concat(path.join('lib', '**', '.test.js')),
```
Comment (iclanton) [inline gulp-core-build-karma/karma.config.js]: This is a really weird convention. Can this be changed to just:

```JavaScript
[bindPolyfillPath, path.join('lib', '**', '.test.js')]
```

Comment (iclanton) [inline gulp-core-build-karma/karma.config.js]: Also - are you sure you want `lib/**/.test.js` as the glob path? Shouldn't it be `lib/**.test.js`?

Comment (dzearing) [inline gulp-core-build-karma/karma.config.js]: Seems fine, but also, we shouldn't hardcode the lib folder:

```ts
let build = require('@microsoft/web-library-build');
let buildConfig = build.getConfig();

// .. and then later:

    // list of files / patterns to load in the browser
    files: [bindPolyfillPath, path.join(buildConfig.libFolder, '**', '*.test.js')],
```

Comment (nickpape-msft) [inline gulp-core-build-karma/karma.config.js]: Good catches, all.

**Pull request closed without merging**